ADAL/src/request/ADWebAuthResponse.m

```diff
@@ -414,6 +414,25 @@ - (void)handleNSError:(NSError*)error
     }
     
     MSID_LOG_WARN(_request, @"System error while making request");
+    
+    // Don't put raw url in NSError because it can contain sensitive data.
+    NSMutableDictionary *errorUserInfo = [error.userInfo mutableCopy];
```

Comment:
can you move this code to its own utility? We might want to reuse it somewhere else :)

Reply:
see #1285 and https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/pull/226